iFixit/react-commerce
# Scale iFixit Images from iFixit API Better

## Overview

We had some images failing to load because we didn't have said images at the dimensions we were requesting. It turns out a reason for this possibly happening was that we would scale up from a given width to the next js preset sizes ( so 180px to 256px for example) and then when we would look up our image dimensions from our api we would scale again off of the next js size (so from 256px to 400px possibly). This is obviously wasteful and unwanted so lets fix this by making sure we only scale off of our expected iFixit dimensions when we can.

## QA 

This should fix images in `ProductList` that were having issues (ie https://www.ifixit.com/Parts/iMac_G5_20%22_Model_A1145). It also affects images in `ProductListSetSection` but I'm not sure how to replicate that exactly, and also if it works on the `ProductList` I'm not sure if it matters to check every use. Any other images should not be affected by this change.

Connects: #583
